spyderlib/spyder.py

```diff
@@ -1541,14 +1489,14 @@ def setup_default_layouts(self, index, settings):
                       finder] + plugins,
                      [console_int, console_ext, console_ipy, history]]
                     ],
-                    'width fraction': [0.0,
+                    'width fraction': [0.0,             # column 0 width
                                        0.55,            # column 1 width
-                                       0.0,
+                                       0.0,             # column 2 width
                                        0.45],           # column 3 width
                     'height fraction': [[1.0],          # column 0, row heights
-                                        [1.0],   # column 1, row heights
-                                        [0.55, 0.45],          # column 2, row heights
-                                        [1.0]],   # column 0, row heights
+                                        [1.0],          # column 1, row heights
+                                        [1.0],          # column 2, row heights
+                                        [0.50, 0.50]],  # column 3, row heights

```

Comment:
I would advice against this. I need to better check what is the available height in the application (Total height minus menu bars and toolbars) and then set the ratios to 0.50, 0.50 which would in theory yield equally sized panes.

Adjusting manually to `[0.46, 0.54]` will make it look good in kubuntu, but it will most likely look bad in windows/osx/ubuntu.